tools/UpdateDotnetRuntime.ps1

```diff
@@ -342,7 +342,7 @@ if ($dotnetUpdate.ShouldUpdate) {
         Import-Module "$PSScriptRoot/../build.psm1" -Force
         Import-Module "$PSScriptRoot/packaging" -Force
         Start-PSBootstrap -Package
-        Start-PSBuild -Clean -Configuration Release -CrossGen -InteractiveAuth:$InteractiveAuth
+        Start-PSBuild -Clean -Configuration Release -CrossGen -InteractiveAuth:$InteractiveAuth -ExperimentalFeatureJsonFilePath "$PSScriptRoot/../ExperimentalFeatures.json"
```

Comment:
If the `ExperimentalFeatures.json` file is always at the same build path location, do we need to pass in the file path?

Reply:
Fixed.. always use the files that are checked in